tests/doctests/wfs_MapServerWFSCapabilities.txt

```diff
@@ -101,6 +101,6 @@ Test exceptions
 Lastly, test the getcapabilities method
 
     >>> wfs = WebFeatureService('http://nsidc.org/cgi-bin/atlas_south?', version='1.0')
-    >>> xml = wfs.getcapabilities().read().decode('utf-8')
+    >>> xml = wfs.getcapabilities().read().decode('ISO-8859-1')
     >>> xml.find('<WFS_Capabilities') > 0
```

Comment:
If open GetCapabilities response you can see:

```
<?xml version='1.0' encoding="ISO-8859-1" ?>
<WFS_Capabilities ...
```
It means that Capabilities document encoding is `ISO-8859-1`. So for converting bytes to a string you have to specify document encoding:

```
xml = wfs.getcapabilities().read().decode('ISO-8859-1')
```